packages/ui/src/ui-component/extended/SpeechToText.jsx

```diff
@@ -16,17 +16,27 @@ import { StyledButton } from '@/ui-component/button/StyledButton'
 import { Dropdown } from '@/ui-component/dropdown/Dropdown'
 import openAISVG from '@/assets/images/openai.svg'
 import assemblyAIPng from '@/assets/images/assemblyai.png'
+import localAISVG from '@/assets/images/robot.png'
```

Comment:
there is a LocalAI png we can copy from `packages/components/nodes/chatmodels/LocalAI`